### PR TITLE
Workflow module and Filteration module

### PR DESCRIPTION
**Pull request by Chandrashekhar-Bhavsar**
Hey guys Please review my code

### CODE REVIEW

File: workflow.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+from queries import *
+from workflow import *
+import datetime
+from datetime import datetime
+import logging
+mydb=connect_db()
+cursor=mydb.cursor()
+
+logging.basicConfig(level=logging.DEBUG)
+
```
Comment (Simran2711) [inline workflow.py:16]: Add the description

File: app.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+logging.basicConfig(level=logging.DEBUG)
+
+file = open("myfile.txt", "w")
+
+app = Flask(__name__)
+cors = CORS(app)
+CORS(app, origins='*')
+
+
```
Comment (Simran2711) [inline app.py:23]: Add the description

File: connection.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    except Exception as e:
+        error = {"error" : "Connection with database is failed","config":config}
+        print(error)
+
+if __name__=='__main__':
+    connect_db()
+
+
+
```
Comment (Simran2711) [inline connection.py:31]: Remove the unwanted Space

File: test_GetWorkflowIssue.py
Comment (Simran2711) [inline test_GetWorkflowIssue.py:1]: Mention the description as what test cases are covered on particular functions.

File: test_IssueFilterationMonth.py
Comment (Simran2711) [inline test_IssueFilterationMonth.py:1]: Mention the description as what test cases are covered on particular functions.

File: test_IssueFilterationQuarter.py
Comment (Simran2711) [inline test_IssueFilterationQuarter.py:1]: Mention the description as what test cases are covered on particular functions.

File: test_IssueFilterationWeek.py
Comment (Simran2711) [inline test_IssueFilterationWeek.py:1]: Mention the description as what test cases are covered on particular functions.

File: test_StatusUpdate.py
Comment (Simran2711) [inline test_StatusUpdate.py:1]: Mention the description as what test cases are covered on particular functions.

File: mainDatabase.sql [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+CREATE TABLE comments (comment_ID INT auto_increment PRIMARY KEY,ID INT NOT NULL,description VARCHAR(100) NOT NULL,date date NOT NULL);
+
+CREATE TABLE workflow (
+  workflow_name VARCHAR(20) NULL DEFAULT NULL,
+  prev_state VARCHAR(20) NULL DEFAULT NULL,
+  current_state VARCHAR(20) NULL DEFAULT NULL,
+  next_state VARCHAR(20) NULL DEFAULT NULL);
+
+
+
```
Comment (pratiksatham) [inline mainDatabase.sql:157]: use pep 8 standard

File: test_GetWorkFlowApi.py
[the remark this answers is not in the document]
Comment (Chandrashekhar-Bhavsar) [reply test_GetWorkFlowApi.py:1]: OK

File: connection.py (hunk 3 above)
Comment (Chandrashekhar-Bhavsar) [reply connection.py:31]: OK NOTED

File: test_GetWorkflowIssue.py
Comment (Chandrashekhar-Bhavsar) [reply test_GetWorkflowIssue.py:1]: ALREADY MENTIONED

File: mainDatabase.sql (hunk 4 above)
Comment (Chandrashekhar-Bhavsar) [reply mainDatabase.sql:157]: Ok will do it

File: app.py (hunk 2 above)
Comment (Chandrashekhar-Bhavsar) [reply app.py]: changes are made

File: test_IssueFilterationMonth.py
Comment (Chandrashekhar-Bhavsar) [reply test_IssueFilterationMonth.py:1]: described

**Review by pratiksatham — APPROVED** [review]

File: Filter.py [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+
+from datetime import datetime, timedelta
+
+# Get the current date
+
+
+
+def IssueFilterationMonth():
+    try:
+        current_date = datetime.now()
```
Comment (Srishtirupa3) [inline Filter.py:26]: Please add doc string

File: test_GetWorkflowIssue.py [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+        # Send a POST request to the API endpoint to add the new user
+        r = requests.post(url=API_ENDPOINT_post, data=json.dumps(data), headers=headers)
+        # Print the response status code
+        print("response error code we got",r.status_code)
+        # Get the response content as JSON
+        Response_Content = r.json()
+        # Print the JSON response
+        print("response content we got",Response_Content)
+        print(" ")
+        # Assert that the response status code is 200 (OK)
```
Comment (Srishtirupa3) [inline test_GetWorkflowIssue.py:76]: Update this comment for code 400

**Review by Simran2711 — APPROVED** [review]

**Review by pratiksatham — APPROVED** [review]

**Review by Srishtirupa3 — APPROVED** [review]
Do the necessary changes as mentioned in the comments.

**Pull request closed without merging**